go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/TecharoHQ/anubis
 
-go 1.24.2
+go 1.24
```

Comment:
👍🏼 

Go's new way to function with the go directive here is a huge pain as most tools refuse to do anything if you are on an older version like 1.24.1